### ISSUE TITLE
hard crash with digital output and duration=61

### ISSUE THREAD

**Issue by bjarthur** [JaneliaSciComp/Wavesurfer#30]
steps to reproduce:

1. add a digital output line to the machine data file as so:

```
physicalOutputChannelNames = {'Dev3/ao0' 'Dev3/line0' 'Dev3/line1'} ;  % Cell array of strings, each string an NI physical channel name
outputChannelNames = {'electrode' 'LED' 'velocity'};  % String cell array of channel identifiers. If left empty, default NI channel names will be used.
```

2. enable stimulation and display

3. set Wavesurfer / Trial Duration and Stimulus Library / Duration to 61 

4. click play

removing the digital output line OR leaving the duration as the 1 sec default alleviates the problem.  they are both necessary

**Comment by bjarthur** [comment]
i should clarify that only one of the digital lines need be removed for the crash to go away.  so seemingly it's a combination of two digital outputs and a duration of 61

**Comment by adamltaylor** [comment]
This seems like it's likely a bug in DABS.  I'm in the process of getting a build environment set up so that I can hopefully debug it.

**Comment by adamltaylor** [comment]
Created an SI issue for this:

http://jira.vidriotechnologies.com/browse/SCIM-183314

**Comment by adamltaylor** [comment]
Fixed in develop branch, commit 87050c35b67b15047b05aa681173858538173637
